### PR TITLE
fix(content): show the reveal banner on replies only

### PR DESCRIPTION
**Pull request by bjoernbg**
The "Reveal originals" bar showed up on things that aren't model replies: the user's own message on ChatGPT, every line of a user query on Gemini, and a conversation title in Gemini's sidebar.

## Cause

Besides the banner on each reply the adapter finds, a fallback puts a banner above any block holding a resolvable token outside those replies. It exists for when a site's reply selectors stop matching, but it ran all the time. Gemini renders each line of a query as its own block, hence one bar per line.

## Fix

- The fallback runs only when `getResponseElements()` returns nothing. Once replies match, banners it placed earlier are removed, restoring the original text first if one was revealed (new `AttachedBanner.detach()`).
- When the fallback does run, it skips:
  - the user's own messages, from a new optional `SiteAdapter.getUserMessageElements()` (ChatGPT: `[data-message-role="user"]`, `[data-message-author-role="user"]`; Claude: `[data-testid="user-message"]`, `.font-user-message`; Gemini: `user-query`, `.user-query-container`)
  - sidebars, headers and navigation (`nav`, `aside`, `header`, `footer` and the matching ARIA roles)

## Testing

- `npx jest`: all suites pass. New tests cover skipping user messages and navigation, detaching a revealed banner, and finding the user message in each site's DOM fixture.
- `npx tsc --noEmit` and `npm run build:ext` pass.
- Tested manually on the live sites. One known gap: if Gemini's sidebar uses none of the tags or roles above, a title there could still get a bar, but only on a page with no replies yet.

### CODE REVIEW

File: src/content/content-script.ts [hunk 1]
```diff
@@ -535,48 +538,71 @@ function reportUnresolvableTokens(seen: readonly string[]): void {
...
+ * outside them is then in the user's message, a sidebar title or similar, not
+ * in a reply. So the fallback stays off, and its banners from before the first
+ * reply matched are removed.
  */
 function attachLooseBanners(composer: HTMLElement | null, transcript: string): void {
+  const turns = adapter.getResponseElements();
+  if (turns.length > 0) {
+    detachLooseBanners();
+    return;
+  }
```
[the remark this answers is not in the document]
Comment (bjoernbg) [reply src/content/content-script.ts:559]: Intentional trade-off. Keeping the fallback on and relying only on the user-message exclusions would bring the original bug back as soon as those selectors break, which is the situation the fallback exists for. Replies in two formats on one page are unlikely, since one page is drawn by one version of the site. After a redesign, nothing matches and the fallback comes back on. The failure here is a reply with no reveal button, not a leak. Model output outside the reply element (artifact or canvas panels) is better covered by adding those selectors to the adapters, in a separate change.
